dataactvalidator/scripts/load_location_data.py

```diff
@@ -297,19 +297,23 @@ def load_location_data(force_reload=False):
     """
     if CONFIG_BROKER["use_aws"]:
         s3_client = boto3.client('s3', region_name=CONFIG_BROKER['aws_region'])
-        city_file = s3_client.generate_presigned_url('get_object', {'Bucket': CONFIG_BROKER['sf_133_bucket'],
-                                                                    'Key': "NationalFedCodes.txt"}, ExpiresIn=600)
-        county_file = s3_client.generate_presigned_url('get_object', {'Bucket': CONFIG_BROKER['sf_133_bucket'],
-                                                                      'Key': "GOVT_UNITS.txt"}, ExpiresIn=600)
-        state_file = s3_client.generate_presigned_url('get_object', {'Bucket': CONFIG_BROKER['sf_133_bucket'],
-                                                                     'Key': "state_list.txt"}, ExpiresIn=600)
+        city_file = s3_client.generate_presigned_url('get_object',
```

Comment:
This is a question for all of these. If they're going in a public bucket, is there any reason we need to separate this and the local versions? Do we have to generate presigned URLs at all or can we just get the file like we do with the agencies files and skip boto for everything except ctystate?